src/streamlink_cli/argparser.py

```diff
@@ -588,6 +588,10 @@ def build_parser():
         Write stream data to FILENAME instead of playing it.
 
         You will be prompted if the file already exists.
+
+        The formatting variables available for the -t/--title option may be used.
+        Invalid characters (cross platform) are replaced with an underscore.
+        Filenames are truncated to 255 characters (cross platform limit).
```

Comment:
> `-t/--title`

There need to be spaces around the slash, otherwise this won't be turned into a link/anchor in the rendered HTML docs.

> 255 characters (cross platform limit)

If `max_len` is set to `None`, then it'll use platform specific defaults instead of 255:
- https://github.com/thombashi/pathvalidate/blob/v2.4.1/pathvalidate/_base.py#L52-L60
- https://github.com/thombashi/pathvalidate/blob/v2.4.1/pathvalidate/_base.py#L85-L95

Reply:
Removing the shorthand `-t` parameter is probably better.

Reply:
> If max_len is set to None, then it'll use platform specific defaults instead of 255:

That's good to know, I had not seen that in the docs.